components/2024/top/access.tsx

```diff
@@ -0,0 +1,40 @@
+import { headlines } from "@/constants/2024/css/css";
+import { Container, Heading, Link, Text, VStack } from "@chakra-ui/react";
+
+export function Access(props: { isPc: boolean }) {
+  const contentFontSize = props.isPc ? "md" : "sm";
+
+  return (
+    <Container maxW="container.xl">
+      <VStack>
+        <Heading className={headlines} as="h2" size="2xl">
+          Access
+        </Heading>
+        <Text fontSize={contentFontSize} textAlign="center">
+          渋谷フクラス 16階にて開催いたします。
+          当日は会場にて飲食の提供はありません。
+        </Text>
+        <Text fontSize={contentFontSize} textAlign="center">
+          また、イベント会場への再入場が難しいため、入場前にお昼ご飯の用意を各自でお願いします。
```

Comment:
```suggestion
          また、イベント会場への再入場ができないため、入場前にお昼ご飯の用意を各自でお願いします。
          会場内でペットボトルのお水の配布は予定していますが、それ以外の飲料は事前にご購入ください。
```

@dorayakikun 個人的には「難しい」ではなく、もはや「できない」としたいかなと思いました。例外がありそうなニュアンスですが、例外があるわけではないので、もう「できない」としておきたいです。

Reply:
水を配る予定なので、それは付記してもよさそうと思いました！